festim/temperature/temperature_solver.py

```diff
@@ -223,7 +223,6 @@ def define_variational_problem(self, materials, mesh, dt=None):
     def define_newton_solver(self):
         """Creates the Newton solver and sets its parameters"""
         self.newton_solver = f.NewtonSolver(f.MPI.comm_world)
-        self.newton_solver.parameters["error_on_nonconvergence"] = False
```

Comment:
Should we explicitly set this to True (even though it's the default value) just for readability?

Reply:
Sure. For extrinsic traps as well?

Reply:
yep